domain-specific-datasets/app/DATASET_README_BASE.md

```diff
@@ -0,0 +1,6 @@
+# Domain Dataset Grower
```

Comment:
I think the wordplay is fun but maybe due to the seed, farming etc is a bit difficult to get what is the core of the project and what is the fun example with a pun, when you try to learn what the project is about.